distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -59,6 +59,14 @@ prepare_visualization y max_rows=1000 =
    Column Limit
 max_columns = 250
 
+##PRIVATE
+whitespace_count : Column -> Integer | Nothing
+whitespace_count col =
+    find_whitespace col = 
+        filtered = col.to_vector.filter (c-> c!=Nothing && (c.to_text.starts_with ' ' || c.to_text.ends_with ' '))
```

Comment:
1. What about other kinds of whitespace? `\t`, `\n` etc. are also whitespace but they do not seem to be considered here.

2. Is the `to_text` needed here? If `col.value_type.is_text` then all values are text already; if `col.value_type == Value_Type.Mixed` then do we want to really look at text representations of various types? Shouldn't we be looking only at the text values and ignore the non-text entries altogether. (i.e. I'm suggesting if maybe we should add `c.is_a Text` to the condition, instead of `.to_text`)

3. Additionally, I think it may make sense to have a 'vectorized' operation for our in-memory columns implemented, as it likely could be more efficient. But I guess it's ok to keep it like this for now and possibly implement in future iterations.